cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -648,19 +677,39 @@ def sub_gate(self) -> RGate:
         return self._sub_gate
 
     @property
-    def phase_exponent(self) -> cirq.TParamVal:  # pylint: disable=missing-function-docstring
+    def phase_exponent(self) -> cirq.TParamVal:
+        """`phase_exponent` property of `ParallelRGate`.
+
+        Returns:
+            `phase_exponent` value of `ParallelRGate`.
+        """
```

Comment:
imo all the properties in this gate should make clear that each of the parallelized RGates share this property, e.g. something like
```suggestion
        """The `phase_exponent` property of each `RGate`."""
```